modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/CassandraPagesRepository.java

```diff
@@ -226,6 +226,27 @@ public ExperimentPageList getExperimentPages(Experiment.ID experimentID) {
     }
 
     @Override
+    public List<PageExperiment> getExperimentsWithoutLabels(Application.Name applicationName, Page.Name pageName) {
+        Stream<PageExperimentByAppNamePage> resultList = Stream.empty();
+        try {
+            Result<PageExperimentByAppNamePage> result = pageExperimentIndexAccessor.selectBy(applicationName.toString(), pageName.toString());
+            resultList = StreamSupport.stream(Spliterators.spliteratorUnknownSize(result.iterator(), Spliterator.ORDERED), false);
+        } catch (ReadTimeoutException | UnavailableException | NoHostAvailableException e) {
+            throw new RepositoryException("Could not retrieve the experiments for applicationName:\"" + applicationName + "\", page:\"" + pageName, e);
```

Comment:
Changed to use StringBuilder... Note: This implementation was done similar to the other methods in the same class and so copied and then modified and so other methods are still doing string concatenation...